cscs-checks/prgenv/cuda/cuda_samples.py

```diff
@@ -8,14 +8,18 @@
 import reframe.utility.osext as osext
 
 
-class CudaSamples(rfm.RegressionTest):
-    def __init__(self):
+@rfm.parameterized_test(['deviceQuery'], ['concurrentKernels'],
+                        ['simpleCUBLAS'], ['bandwidthTest'],
+                        ['conjugateGradientCudaGraphs'])
+class CudaSamplesTest(rfm.RegressionTest):
+    def __init__(self, test_name):
```

Comment:
I'd change this to use the new parameter syntax instead
```suggestion
@rfm.simple_test
class CudaSamplesTest(rfm.RegressionTest):
    parameter('test_name', ['deviceQuery', 'concurrentKernels', 
        'simpleCUBLAS', 'bandwidthTest', 
        'conjugateGradientCudaGraphs']
    )
    def __init__(self):
```